### DISCUSSION TITLE
How do I change vertical fillchars on Lunarvim?

### DISCUSSION THREAD

**Discussion by nikolasetiawan** [LunarVim/LunarVim#2262] [Q&A] [answered]
Hi, I'm having quite a problem trying to change the vert fillchars on Lunarvim.

I have tried lua way:
`vim.opt.fillchars = vim.opt.fillchars + "vert:\ "`
and vim way
`vim.cmd([[set fillchars=vert:\ ]])`

Both with no success. Have anybody had luck with this?

**Comment by kylo252** [comment, marked as answer]
```lua
vim.opt.fillchars = {
  fold = " ", -- remove folding chars
  vert = "\\", -- set vsplit chars
}
```
